tasks/publish.yaml

```diff
@@ -1,16 +1,44 @@
 variables:
   - name: TARGET_REPO
     default: ghcr.io/defenseunicorns/packages/uds
+  - name: FLAVOR
+    default: upstream
 
 tasks:
   - name: package
-    description: Publish the packages
+    description: Publish packages for the amd64 and arm64 architectures
+    inputs:
+      path:
+        description: Path to the zarf package being published
+        default: .
+      version:
+        description: The version of the package to publish
+        required: true
     actions:
-      - description: Publish the packages
+      - description: Publish packages for the amd64 and arm64 architectures
         cmd: |
           set -e
-          uds zarf package publish zarf-package-*-amd64-${VERSION}.tar.zst oci://${TARGET_REPO}
+          uds zarf package publish ${{ .inputs.path }}/zarf-package-*-amd64-${{ .inputs.version }}.tar.zst oci://${TARGET_REPO}
           if [ ${FLAVOR} != "registry1" ]; then
-            uds zarf package publish zarf-package-*-arm64-${VERSION}.tar.zst oci://${TARGET_REPO}
+            uds zarf package publish ${{ .inputs.path }}/zarf-package-*-arm64-${{ .inputs.version }}.tar.zst oci://${TARGET_REPO}
+          fi
+          set +e
+
+  - name: bundle
+    description: Publish bundles for the amd64 and arm64 architectures
+    inputs:
+      path:
+        description: Path to the UDS bundle being published
+        default: bundle
+      version:
+        description: The version of the bundle to publish
+        required: true
+    actions:
+      - description: Publish bundles for the amd64 and arm64 architectures
+        cmd: |
+          set -e
+          uds publish ${{ .inputs.path }}/uds-bundle-*-amd64-${{ .inputs.version }}.tar.zst oci://${TARGET_REPO}/bundles --no-progress
```

Comment:
I think im totally fine with this being either way, but in line with making the version an input do you want to do the same with TARGET_REPO?

Reply:
No is a totally valid answer 🤣

Reply:
So is YES :sweat_smile: